### DISCUSSION TITLE
what is gitignore? Are there any sources available for useful .gitignore files for projects?

### DISCUSSION THREAD

**Discussion by netdevnam** [community/community#28010] [Other Feature Feedback, Questions, & Ideas] [answered]
what is gitignore? Are there any sources available for useful .gitignore files for projects?

**Comment by truongletann** [comment, marked as answer]
A gitignore file specifies reluctantly untracked files that Git should ignore. 
Currently there are many sources but I often use it at: https://www.toptal.com/developers/gitignore . You can refer to the link above
